playground/src/editor.tsx

```diff
@@ -8,12 +8,12 @@ import {
   CompilerState,
   ICompilerWorker,
   ILanguageServiceWorker,
-  LanguageServiceEvent,
   QscEventTarget,
   VSDiagnostic,
   log,
   ProgramConfig,
   TargetProfile,
+  LanguageServiceDiagnosticEvent,
```

Comment:
With this rename it'll likely be a breaking change for QCOM. We'll need to ensure we communicate this when they take a new drop.

Reply:
It is wrapped in a type of the same name as before, this is just one variant of the sum type -- @minestarks [mentioned that it isn't breaking for QCOM](https://github.com/microsoft/qsharp/pull/2095#discussion_r1906211203) here. Are you sure this is breaking? Happy to communicate it either way.

Reply:
Oh crap I'm sorry. I missed that this might trigger a TypeScript error, even though it's not a breaking change at runtime. Should be straightforward to update QCOM though.